mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -151,6 +155,29 @@ class ShowArtifactLoggedModelView extends Component<Props, State> {
     );
   }
 
+  validateModelForServingText(modelPath: any, servingInput?: string) {
+    if (servingInput === null) {
+      return `from mlflow.models import validate_serving_input\n
```

Comment:
```suggestion
      return `from mlflow.models import validate_serving_input
```

Is `\n` necessary?

Reply:
I actually want to keep this because we usually leave a blank line between imports and the code to execute

Reply:
yeah i think this is a multiline string so it shouldn't be necessary to explicitly put `\n`